### DISCUSSION TITLE
v0.3.19

### DISCUSSION THREAD

**Discussion by littlesera** [littlesera/LSCG#362] [Announcements]
## What's Changed
* actually fix regression >.> by @littlesera in https://github.com/littlesera/LSCG/pull/359
* Merge Changes from Dev by @littlesera in https://github.com/littlesera/LSCG/pull/361
  * Version bump to v0.3.19
  * Add whitelist-only magic setting
  * Add fellatio activities
  * Add legs+feet nuzzling
  * Add breb eating
  * Fix #335 
  * Fix 'ride with--' activitity still allowed when clothed 

**Full Changelog**: https://github.com/littlesera/LSCG/compare/v0.3.18...v0.3.19

<hr /><em>This discussion was created from the release <a href='https://github.com/littlesera/LSCG/releases/tag/v0.3.19'>v0.3.19</a>.</em>

**Comment by anchillabc** [comment]
Might I humbly suggest adding the breb eating to the mouth slots as well?

**Reply by littlesera** [reply]
It was on the hand to facilitate eating other people's brebs, but after backing off that idea (offload breb-eating fight to "steal") I think I'll just move it to the mouth in general.

**Reply by anchillabc** [reply]
Ah makes sense.
The only people around me only ever have burned breb, so it never occurred to me to eat theirs. Why not both though?

**Reply by littlesera** [reply]
To keep it more aligned with base BC. You can't "sip" someone else's held drink either, which is a standard activity. Also to keep at least a small barrier to prevent someone else from using the effect in your bread if you don't want. They would have to 'steal' it from you, which is an activity you can block (opt-out of)
